federation/ipatuura/README.md

```diff
@@ -172,12 +172,18 @@ The `integrationdomain.json` file looks like this:
    "id_provider": "ad",
    "user_extra_attrs": "mail:mail, sn:sn, givenname:givenname",
    "user_object_classes": "user,organizationalPerson,person,top",
-   "users_dn": "CN=Keycloak,CN=Users,DC=keycloak,DC=org",
+   "users_dn": "CN=Users,DC=keycloak,DC=org",
```

Comment:
This `CN=Keycloak` OU does not exist by default so the bridge will fail when trying to add new users with
```
[Tue Jan 21 16:40:11.347801 2025] [wsgi:error] [pid 65:tid 168] [remote 10.88.0.1:59778] LDAP Error: No such object: 00002030: objectclass: Cannot add cn=newuser7,CN=Keycloak,CN=Users,DC=keycloak,DC=org, parent does not exist!
```

Reply:
@justin-stephenson yeah, good catch - I used my local setup where I keep users in a different DN to avoid mixing Keycloak users with default AD users that already exist in the main DN. Documentation should use the default DN though.